src/core/env/UserContext.tsx

```diff
@@ -0,0 +1,15 @@
+import { ZetkinUser } from 'utils/types/zetkin';
+import { createContext, FC, ReactNode } from 'react';
+
+const UserContext = createContext<ZetkinUser | null>(null);
```

Comment:
There is already a UserContext, created in the file useFocusDate.ts (which is a weird place to have a user context be created but i guess there was at some point a reason for this desicion), is it intentional to have it created in two places? It's used in the Providers file. 

Should the old one be deleted maybe, and the use in Providers be replaced with this one?

Reply:
@richardolsson since this is already merged, should we make a separate issue for handling the two `UserContext`s?